public/js/helpers/chartsHelpers.js

```diff
@@ -98,4 +98,60 @@ const downloadCSV = (data, chartType, filterVals) => {
     }
 };
 
-export { createPartialsList, formattedSeries, createYTotalsList, compareDates, compareIssueDates, fillMissingDates, downloadCSV, tagToName };
\ No newline at end of file
+const tidyData = (data, startDate, endDate) => {
```

Comment:
From the name it's not really clear what the function does. I'd go for a longer more explicit name